generators/app/templates/app/middlewares/docsAuth.ejs

```diff
@@ -0,0 +1,19 @@
+const { decode } = require('jwt-simple');
+
+const logger = require('../logger');
+
+const isDocumentationAuthenticated = async (req, res, next) => {
```

Comment:
since there is only one function in this file I would prefer `exports.isDocumentationAuthenticated` or `module.exports = { (req, res, next) => { .... } }`